projects/components/src/toggle-switch/toggle-switch.component.scss

```diff
@@ -3,8 +3,14 @@
 
 :host {
   ::ng-deep {
+    .mat-slide-toggle {
+      height: 20px;
+      line-height: 20px;
+    }
+
     .mat-checked {
       .mat-slide-toggle-thumb-container {
+        transform: translate3d(10px, 0, 0) !important;
```

Comment:
is the `!important` needed here? We generally want to save that for a tool of last resort

Reply:
I checked the material documentation and did not find a better solution, I know that it's not the best solution.

Reply:
@palbizu do we want this smaller dimension toggles everywhere? Could you please check with Michael about this? If not, we'll have to introduce a size param in the component.

Reply:
@arjunlalb It was validated with Michael the changes effectively apply to all toggle buttons